diff-models/core/jest.config.js

```diff
@@ -0,0 +1,24 @@
+export default {
```

Comment:
Almost 100% certain this isn't needed. Can you remove all boilerplate that you don't have a specific purpose for from this folder?

Reply:
Good point to remove unnecessary files, but this one is very much needed. Jest will not run without this.